### ISSUE TITLE
[FEATURE] Profile Pictures

### ISSUE THREAD

**Issue by ThePhaseCat** [trainb0y1/PyMultibound#7]
I think it would be interesting (and cool) if each profile can have a "profile picture" to better identity different profiles. I don't even know if this was possible. This would also require some way to select custom ones saved on the user's computer, which could maybe use a similar UI to #5 

Example:
-A Frackin' Universe pfp for a Frackin' Universe profile
-A Starbound logo pfp for a vanilla profile

(can't put the enhancement label but this is an enhancement in my mind)

**Comment by trainb0y1** [comment]
Definitely possible, but I wouldn’t know a good way to determine the icon for most mods, without the user doing it themselves. 

More pressing things to add at the moment, but this is definitely a cool idea

**Comment by ThePhaseCat** [comment]
Yeah, it can be a side feature in case you want to add something to the application

**Comment by trainb0y** [comment]
Likely not happening unless I decide to rewrite this whole thing in Rust or something idk
(mostly posting this as a test for a discord webhook)
